deploy/role_binding.yaml

```diff
@@ -9,3 +9,15 @@ roleRef:
   kind: Role
   name: smart-gateway-operator
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: smart-gateway-operator
+subjects:
+- kind: ServiceAccount
+  name: smart-gateway-operator
```

Comment:
```suggestion
  name: smart-gateway-operator
  namespace: placeholder
```

Reply:
as in https://github.com/infrawatch/service-telemetry-operator/blob/master/deploy/role_binding.yaml#L8